pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/SimplePlayerViewModel.kt

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2022. SRG SSR. All rights reserved.
+ * License information is available from the LICENSE file.
+ */
+package ch.srgssr.pillarbox.demo.ui.player
+
+import android.app.Application
+import android.util.Log
+import androidx.lifecycle.AndroidViewModel
+import androidx.media3.common.MediaItem
+import androidx.media3.common.MediaMetadata
+import androidx.media3.common.PlaybackException
+import androidx.media3.common.PlaybackParameters
+import androidx.media3.common.Player
+import androidx.media3.common.Timeline
+import ch.srgssr.pillarbox.demo.data.DemoItemDataSource
+import ch.srgssr.pillarbox.demo.data.DemoMediaItemSource
+import ch.srgssr.pillarbox.player.PillarboxPlayer
+
+/**
+ * Simple player view model than handle a PillarboxPlayer [player]
+ */
+class SimplePlayerViewModel(application: Application) : AndroidViewModel(application), Player.Listener {
+    /**
+     * Player as PillarboxPlayer
+     */
+    val player = PillarboxPlayer(application, DemoMediaItemSource(DemoItemDataSource(application)))
+
+    init {
+        player.addListener(this)
+    }
+
+    /**
+     * Add to [player] all [ids] to the MediaItem list.
+     * Will prepare and play the content.
+     *
+     * @param ids mediaIdentifier to play
+     */
+    fun playItemIds(ids: Array<String>) {
+        player.addMediaItems(ids.map { fromMediaId(it) })
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SRGSSR/pillarbox-android/blob/9e672e88664d24e42b2fbb65a69ff4507dff46b6/pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/SimplePlayerViewModel.kt#L40"></span>Access to <code>private</code> method <code>fromMediaId</code> of class <code>Companion</code> requires synthetic accessor</td>
    </tr>
  </tbody>
</table>